src/commands/mod.rs

```diff
@@ -860,7 +860,7 @@ impl DaemonControl {
     pub fn get_spend_tx(
         &self,
         outpoints: &[OutPoint],
-        destinations: BTreeMap<Address, u64>,
+        destinations: &BTreeMap<Address, u64>,
```

Comment:
Hmmm i thought we consumed it. But the `collect()` in
```rust
        let txos: Vec<SpendTxOut> = destinations
            .into_iter()
            .map(|(addr, value)| {
                let script_pubkey = addr.script_pubkey();
                SpendTxOut::new(TxOut {
                    value: *value,
                    script_pubkey,
                })
            })
            .collect();
```
seems to have the effect of a clone.

I'd prefer if we moved it but it's not important.

Reply:
`script_pubkey()` method need only a reference and `value` is a u64: no need to clone the btreemap values and I think `into_iter` does not clone it (I will change for iter()).
I think it is good practice that if your method does not need to own an object to pass only a ref to it

Reply:
Yes it definitely is. As mentioned in my comment, my mistake i figured it had to be moved :)